### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by Moon-DaeSeung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,59 @@
...
+For detailed guides and documentation, visit our [official documentation](https://notionpresso.com).
 
-To learn more about Next.js, take a look at the following resources:
+## Documentation
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+- [Getting Started Guide](https://notionpresso.com/docs/getting-started/introduction)
+- [Tutorial](https://notionpresso.com/tutorial)
+- [Blog Template](https://nextjs-blog-template.pages.dev/)
```
Comment (anonymousRecords) [inline README.md:51]: p3: What do you think about adding a showcase?

https://notionpresso.com/ko/showcase

**Review by anonymousRecords — COMMENTED** [review]
I personally feel that the README for Notion Docs resembles the README of the Notionpresso open source project. 
It seems that the contribution section only discusses contributions related to the Notionpresso open source itself, without mentioning contributions to the documentation. I'm curious about your opinion on this aspect.

Also, when looking at the official documentation repository for pmndrs, it describes how to run the official documentation locally.
https://github.com/pmndrs/docs/blob/main/docs/getting-started/introduction.mdx

File: README.md (hunk 1 above)
Comment (Moon-DaeSeung) [reply README.md:51]: good

Comment (Moon-DaeSeung) [reply README.md:51]: I added it

**Comment by Moon-DaeSeung** [comment]
@anonymousRecords I just wrote it with ai.

I think the we can also develop docs's readme. but it might be time-consuming. 

내용은 반복되어도 좋다고 생각해요 사람들이 어디로 진입할지 모르니 ㅎ, 대신 docs 에 대한 기여방법이 자세히 추가되면 좋겠네요.

**Pull request merged**